### DISCUSSION TITLE
Regarding Units

### DISCUSSION THREAD

**Discussion by dineshkumarreddyg36** [xcompact3d/Incompact3d#103] [Q&A] [answered]
What are the **units** that are considered in this solver? For example in the **cylinder** case the **length** of the computational domain is given **(xlx=50.)**. Could you please tell what are the units, like whether it is in **meters or centimeters or millimeters**?

**Comment by slaizet** [comment]
The code is designed for non-dimensional simulations, except possibly for the wind farm simulator. For the cylinder, the data are normalised with the diameter of the cylinder (D=1) and the freestream velocity (U=1).
In general, you can decide to do non-dimensional or dimensional simulations, as long as your are consistent with all the parameters.

Best,

Sylvain

**Reply by dineshkumarreddyg36** [reply]
In **Cylinder** case, there is one test case **input_test_moving.i3d**  in which the **Reynolds number is given 300**. If I want to run the simulation for **Re 1000**, what **parameters** should I change?

**Comment by slaizet** [comment, marked as answer]
This input file is for a moving cylinder (ambient fluid nearly at rest). You should look at input_DNS300_LR.i3d, which is for a fixed cylinder. To switch from Re=300 to Re=1000, you just need to change the spatial resolution and time step (and of course Re in the input file).

Best,

Sylvain

**Reply by dineshkumarreddyg36** [reply]
ok thank you @slaizet
